src/test/resources/beneficiary_truncated.rif

```diff
@@ -0,0 +1,10 @@
+DML_IND|BENE_ID|STATE_CODE|BENE_COUNTY_CD|BENE_ZIP_CD|BENE_BIRTH_DT|BENE_SEX_IDENT_CD|BENE_RACE_CD|BENE_ENTLMT_RSN_ORIG|BENE_ENTLMT_RSN_CURR|BENE_ESRD_IND|BENE_MDCR_STATUS_CD|BENE_PTA_TRMNTN_CD|BENE_PTB_TRMNTN_CD|BENE_CRNT_HIC_NUM|BENE_SRNM_NAME|BENE_GVN_NAME|BENE_MDL_NAME
```

Comment:
I'd rely on the sample data we've already got, as the RIF schema changes semi-frequently. See here: `bluebutton-data-model-rif-samples/gov.hhs.cms.bluebutton.data.model.rif.samples.StaticRifResource`.

Reply:
If you did make that change, can/should you just remove this file? Or is it still used for something else?

Reply:
I left it for the "local" rif file test.  That is really a test that injects a custom rif file instead of using the sample data classes.  I know those are backed by actual rif files but I thought this might be useful for when we go to S3 backed rif files.  I will probably leverage your S3 rif file implementation for this as well but for now I thought it could stay until we get there.